src/main/python/org/broadinstitute/hellbender/gcnvkernel/postprocess/viterbi_segmentation.py

```diff
@@ -199,27 +200,27 @@ def _viterbi_segments_generator_for_single_sample(self, sample_index: int)\
                 alpha_tc, beta_tc, log_posterior_prob_tc, log_data_likelihood)
 
             # coalesce into piecewise constant copy-number segments, calculate qualities
-            for copy_number_call, start_index, end_index in self._coalesce_seq_into_segments(viterbi_path_t_contig):
+            for call_copy_number, start_index, end_index in self._coalesce_seq_into_segments(viterbi_path_t_contig):
                 num_points = end_index - start_index + 1
                 segment = IntegerCopyNumberSegment(contig,
                                                    contig_interval_list[start_index].start,
                                                    contig_interval_list[end_index].end,
                                                    num_points,
-                                                   copy_number_call,
-                                                   contig_baseline_copy_number_state)
+                                                   call_copy_number,
+                                                   contig_baseline_copy_number)
                 if num_points > 1:
                     segment.quality_some_called = segment_quality_calculator.get_segment_quality_some_called(
-                        start_index, end_index, copy_number_call)
+                        start_index, end_index, call_copy_number)
                     segment.quality_all_called = segment_quality_calculator.get_segment_quality_all_called(
-                        start_index, end_index, copy_number_call)
+                        start_index, end_index, call_copy_number)
                     segment.quality_start = segment_quality_calculator.get_segment_quality_start(
-                        start_index, copy_number_call)
+                        start_index, call_copy_number)
                     segment.quality_end = segment_quality_calculator.get_segment_quality_end(
-                        end_index, copy_number_call)
+                        end_index, call_copy_number)
 
                 else:  # for single-interval segments, all qualities must be the same
                     segment.quality_some_called = segment_quality_calculator.get_segment_quality_some_called(
-                        start_index, end_index, copy_number_call)
+                        start_index, end_index, call_copy_number)
```

Comment:
I think you forgot to change one "export" method below.

Reply:
damn! :D fixed.